src/components/UserActionRequired/index.jsx

```diff
@@ -0,0 +1,111 @@
+/* global __TARGET__, cozy */
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import ReactMarkdown from 'react-markdown'
+import tosIcon from 'assets/icons/icon-tos.svg'
+import { Modal, Icon, Button, translate, Alerter } from 'cozy-ui/react'
+import styles from './styles.styl'
+import { onLogout } from 'ducks/authentication/MobileRouter'
+
+const TosUpdatedModal = translate()(({ t, newTosLink, onAccept, onRefuse }) => (
+  <Modal closable={false}>
+    <Modal.ModalHeader />
+    <Modal.ModalDescription className={styles['tosupdated']}>
+      <Icon icon={tosIcon} width={96} height={96} />
+      <h2 className={styles['tosupdated-title']}>{t('TOS.updated.title')}</h2>
+      <ReactMarkdown
+        className={styles['tosupdated-desc']}
+        source={t('TOS.updated.detail', { link: newTosLink })}
+      />
+      <Button
+        extension="full"
+        label={t('TOS.updated.cta')}
+        onClick={onAccept}
+      />
+      <Button
+        subtle
+        size="small"
+        extension="full"
+        style={{ marginTop: '1.5rem' }}
```

Comment:
`className='u-mt-1-half'` is available if you want

Reply:
:+1: